tests/runtests.py

```diff
@@ -78,12 +78,12 @@ def main() -> None:
     if folder not in {"stdlib", "stubs"}:
         parser.error("Only the 'stdlib' and 'stubs' folders are supported.")
     if not os.path.exists(path):
-        parser.error(rf"'path' {path} does not exist.")
+        parser.error(rf"{path=} does not exist.")
```

Comment:
Nit: The "r" has no function here:

```suggestion
        parser.error(f"{path=} does not exist.")
```